front/components/sparkle/navigation.tsx

```diff
@@ -205,20 +206,32 @@ export const subNavigationBuild = ({
     ],
   });
 
-  if (isDevelopmentOrDustWorkspace(owner)) {
+  const hasBetaAccess = owner.flags?.some((flag: string) =>
+    flag.startsWith("labs_")
+  );
+  const betaMenus: SparkleAppLayoutNavigation[] = [];
+
+  if (hasBetaAccess) {
+    if (
+      owner.flags.includes("labs_extract") ||
+      isDevelopmentOrDustWorkspace(owner)
+    ) {
+      betaMenus.push({
+        id: "extract",
+        label: "Extract",
+        icon: ArrowUpOnSquareIcon,
+        href: `/w/${owner.sId}/builder/labs/extract`,
+        current: current === "extract",
+        subMenuLabel: current === "extract" ? subMenuLabel : undefined,
+        subMenu: current === "extract" ? subMenu : undefined,
+      });
+    }
+
```

Comment:
You likely want to push transcripts here as well, no?

Reply:
No as it's for all users and not builders only it's been moved to user menu